src/consts.ts

```diff
@@ -0,0 +1,11 @@
+export const WEEK_DAYS = ["Mo", "Tu", "We", "Th", "Fr", "Sa", "Su"];
+export const MONTHS = ["Jan", "Feb", "Mar", "Apr", "May", "June", "July", "Aug", "Sept", "Oct", "Nov", "Dec"];
+export const DATE_RANGE_ACTIONS = {
+  today: "Today",
+  yesterday: "Yesterday",
+  thisWeek: "This Week",
+  lastWeek: "Last Week",
+  last7Days: "Last 7 Days",
+  thisMonth: "This Month",
+  lastMonth: "Last Month",
+};
```

Comment:
By adding `as const`, the typescript will do a better type inference. Without it the type would just be `string[]`.

```suggestion
export const WEEK_DAYS = ["Mo", "Tu", "We", "Th", "Fr", "Sa", "Su"] as const;
export const MONTHS = ["Jan", "Feb", "Mar", "Apr", "May", "June", "July", "Aug", "Sept", "Oct", "Nov", "Dec"] as const;
export const DATE_RANGE_ACTIONS = {
  today: "Today",
  yesterday: "Yesterday",
  thisWeek: "This Week",
  lastWeek: "Last Week",
  last7Days: "Last 7 Days",
  thisMonth: "This Month",
  lastMonth: "Last Month",
} as const;
```